src/geometry/rotation_specialization.rs

```diff
@@ -979,6 +979,113 @@ impl<T: SimdRealField> Rotation3<T> {
             )
         }
     }
+
+    /// Represent this rotation as Euler angles.
+    ///
+    /// Returns the angles produced in the order provided by seq parameter, along with the
+    /// observability flag. If the rotation is gimbal locked, then the observability flag is false.
+    ///
+    /// Algorithm based on:
+    /// Malcolm D. Shuster, F. Landis Markley, “General formula for extraction the Euler
+    /// angles”, Journal of guidance, control, and dynamics, vol. 29.1, pp. 215-221. 2006,
+    /// and modified to be able to produce extrinsic rotations.
```

Comment:
Could you please add a couple of examples (as doc-tests) here? Perhaps the ones presented in the papier.
In addition, does `seq` have to match some properties (being an orthonormal basis)? If so, that should be mentioned in the documentation here.

Reply:
Yes, the axes must be orthonormal (satisfying n1 dot n2 = 0 and n3 dot n1 = 0). I added asserts for that as well